goldenrun.py

```diff
@@ -82,6 +80,8 @@ def run_goldenrun(
                 f"{experiment['type']} not finished. Probably no valid instruction! "
                 f"If valid increase tb max for golden run"
             )
+        else:
```

Comment:
I think we can drop the `else` statement here, as within the `if` block a ValueError is raised.

Reply:
Yep, makes sense. I just pushed an updated version.